### PR TITLE
Added single video page

### PR DESCRIPTION
**Pull request by tanveertkd**
This PR adds the single video page feature. You can watch a video by clicking on it.

### CODE REVIEW

File: src/context/like-context.js [hunk 1]
```diff
@@ -12,16 +12,20 @@ const LikeProvider = ({ children }) => {
...
+                }));
+                toast.success('Added to liked videos!');
+            } else {
+                toast.error("Couldn't add to liked videos!");
+                console.log(response.data.errors);
+            }
         }else{
-            toast.error("Couldn't add to liked videos!");
-            console.log(response.data.errors);
+            toast.error("You must be logged in to like a video.");
```
Comment (himadri2110) [inline src/context/like-context.js:28]: Suggestion: Here you can redirect the user to login/signup page

File: src/services/videos-services.js [hunk 2]
```diff
@@ -3,9 +3,20 @@ import axios from "axios";
...
 }
 
-export { getVidoes };
\ No newline at end of file
+const getVideoById = async (videoId) => {
+    try{
+        const response = await axios.get(`/api/video/${videoId}`);
+        if(response.status === 200){
+            return response;
+        }
```
Comment (himadri2110) [inline src/services/videos-services.js:16]: Suggestion: You can make a service to get a particular video by its ID.

**Review by himadri2110 — APPROVED** [review]
The single video page looks clean. Good to go.

**Review by poornima-alugubelly — APPROVED** [review]
Good work 👍

**Pull request merged**